test-app/tests/integration/components/headless-form-validation-test.gts

```diff
@@ -2030,6 +2030,106 @@ module('Integration Component HeadlessForm > Validation', function (hooks) {
       assert.dom('[data-test-invalid]').doesNotExist();
     });
 
+    test('form yields rawErrors', async function (assert) {
+      const data: TestFormData = { firstName: 'foo' };
+
+      await render(<template>
+        <HeadlessForm
+          @data={{data}}
+          @validate={{validateFormCallbackSync}}
+          as |form|
+        >
+          {{#each-in form.rawErrors as |name errors|}}
```

Comment:
👀 ! I didn't know about `{{each-in}}`. Looks useful!

Reply:
Yeah, so it _might_ actually be the first time I used it in my idk ~10y of working w/ Ember! 😅. (and maybe I just forgot about it, but you definitely don't see very often)